src/xrSound/SoundRender_Core.cpp

```diff
@@ -115,7 +115,7 @@ CSound* CSoundRender_Core::create(pcstr fName, esound_type sound_type, int game_
     auto* snd = xr_new<CSound>(handle);
 
     snd->g_type = game_type;
-    if (game_type == sg_SourceType)
+    if (game_type == static_cast<int>(sg_SourceType))
```

Comment:
`game_type` is `u32`, so no need to cast `sg_SourceType` to `int`.
```suggestion
    if (game_type == sg_SourceType)
```